docs/api-irma-server.md

```diff
@@ -182,30 +188,64 @@ Behaves exactly the same as the [delete endpoint for requestors](#delete-session
 
 ### `GET /irma/session/{clientToken}/status`
 Behaves exactly the same as the [status endpoint for requestors](#get-session-requestortoken-status), but uses the [client token
-from the `sessionPtr`](#post-session) instead of the requestor token.
+from the `sessionPtr`](#post-session) instead of the requestor token. For frontend libraries, this endpoint is deprecated.
+Please use the [frontend status endpoint](#get-irma-session-clienttoken-frontend-status) instead.
 
 ---
 
 ### `GET /irma/session/{clientToken}/statusevents`
 Behaves exactly the same as the [statusevents endpoint for requestors](#get-session-requestortoken-statusevents), but uses the
-[client token from the `sessionPtr`](#post-session) instead of the requestor token.
+[client token from the `sessionPtr`](#post-session) instead of the requestor token. For frontend libraries this endpoint is deprecated.
+Please use the [frontend statusevents endpoint](#get-irma-session-clienttoken-frontend-statusevents) instead.
 
 ---
 
+## API reference `/irma` frontend endpoints
+The frontend endpoints are exclusively meant for [frontend libraries](irma-frontend.md) to communicate with the IRMA server.
+Frontends need the information from the `frontendRequest` in order to use these endpoints. The `frontendRequest` is received
+along with the `sessionPtr` from the [`POST /session`](#post-session) requestor endpoint.
+Just like the [other `/irma` endpoints](#api-reference-irma-endpoints), the `/irma/frontend` endpoints of your IRMA server
```

Comment:
Here is a last occurence of `/irma/frontend`.

Reply:
Right, missed that one. Fixed it now.